### PR TITLE
Community Stuff 

### LINKED ISSUE #8: Improve user messages
Before this project enter on Alpha, we need create a way to distribute releases :) I think in use github:tags for that, like [apex](https://github.com/apex/apex) do :D

I think that we need to improve user messages in core of program, to explain better what the commands do :D

( X ) Create a Release Distribution
( ) Improve User Messages
( ) Create Contribution Guide / Code Conduct (and other stuffs)
( ) Finish README.md

### PR DESCRIPTION
**Pull request by guidiego**
### Issue
this resolve #8 

### Description
- Create Community Stuff

### Topics (optional)
- Added `.github` folder with community stuff (ok)
- Final Version Of Readme (ok)
- Improve User Messages (ok)

### Any photo of a cute animal or a beautiful place! (not mandatory but enconraged)
img

### CODE REVIEW

**Comment by guidiego** [comment]
@joaoh82 the enum that we talked about today can enter on this PR with the new improved messages! What you think?

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Purrgil follows [semantic versioning](http://semver.org/). We release patch versions for bugfixes, minor versions for new features, and major versions for any breaking changes. When we make breaking changes, we also introduce deprecation warnings in a minor version so that our users learn about the upcoming changes and migrate their code in advance.
+
+### Bugs
+
+#### Where to Find Known Issues
+
+We are using [GitHub Issues](https://github.com/purrgil/purrgil/issues) for our public bugs. We keep a close eye on this and try to make it clear when we have an internal fix in progress. Before filing a new task, try to make sure your problem doesn't already exist.
+
+#### Dicussions
```
Comment (joaoh82) [inline .github/CONTRIBUTING.md]: "Discussions"

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+If you want to init a dicussion about something releated with Purrgil, create issue with `[FORUM]` in start of the title! We have some forums opened by core team :)
```
Comment (joaoh82) [inline .github/CONTRIBUTING.md]: "If you want to init a discussion"

Comment (joaoh82) [inline .github/CONTRIBUTING.md]: "create an issue with"

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+### About what we are going talk?
```
Comment (joaoh82) [inline .github/ISSUE_TEMPLATE.md]: "Are we going to talk about what"

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+Bug, Feature, Backlog Candidate, Improvement, Question or anything
+
+### Short Description
+Tell us what you want/need or send us draft :joy:
+
+### Tecnical Details
```
Comment (joaoh82) [inline .github/ISSUE_TEMPLATE.md]: "Technical"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+### Issue
+> If your PR are releated with some issue please use [resolve command](https://github.com/blog/1506-closing-issues-via-pull-requests) in this section! ex:
+> `this resolve #1`
+
+### Decription
```
Comment (joaoh82) [inline .github/PULL_REQUEST_TEMPLATE.md]: "Description"

**Review by joaoh82 — COMMENTED** [review]
Added several comments on the text referring to words spelled wrong.

**Comment by joaoh82** [comment]
@guidiego I think that a PR usually attacks a specific problem. Maybe we could do this PR with the texts and the do another one with what we talked about.

**Pull request merged**